__tests__/api.test.js

```diff
@@ -91,8 +91,25 @@ describe('GET /api/articles', () => {
             expect(articles).toBeSortedBy('created_at', {descending: true})
         })
     })
+    test('when queried with a topic value it responds status code 200 and with all article of that topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({ body }) => {
+            const articles = body.articles
```

Comment:
what if the topic=paper which id valid but there are no articles

Reply:
i think that is what i meant by invalid topic, does this test cover that? 
Ive returned an empty array
Ive altered the wording of the test.